src/sdc11073/sdcclient/sdcclientimpl.py

```diff
@@ -411,15 +412,18 @@ def localization_service_client(self):
     def subscription_mgr(self):
         return self._subscription_mgr
 
-    def start_all(self, not_subscribed_actions=None,
-                  subscriptions_check_interval=None,
-                  subscribe_periodic_reports=False,
-                  shared_http_server=None):
+    def start_all(self, not_subscribed_actions: Optional[list[str]] = None,
+                  subscriptions_check_interval: Optional[float] = None,
+                  subscribe_periodic_reports: bool = False,
+                  shared_http_server: Optional[Any] = None,
+                  check_get_service: bool = True) -> None:
         """
         :param not_subscribed_actions: a list of pmtypes.Actions elements or None. if None, everything is subscribed.
         :param subscriptions_check_interval: an interval in seconds or None
         :param subscribe_periodic_reports:
         :param shared_http_server: if provided, use this http server, else client creates its own.
+        :param check_get_service: if True (default) it checks that a GetService is detected,
+               which is the minimal requirement for a sdc provider.
```

Comment:
is it the only minimal requirement for a sdc provider?

Reply:
as far as I know: yes